tests/test_climate_baseline_helpers.py

```diff
@@ -57,23 +49,23 @@ def test_creates_the_expected_new_network_record(empty_database_session):
         assert result.publish == True
         assert 'PCIC' in result.long_name
 
-    def test_creates_no_more_than_one_of_them(empty_database_session):
-        sesh = empty_database_session
+    def it_creates_no_more_than_one_of_them(session):
+        sesh = session
```

Comment:
This is covered in PR #10. But it doesn't cover such cases, probably, and there are other little inconsistencies. How about we leave this aside and I will create a cleanup PR for the leftovers.